src/components/Postiterie.vue

```diff
@@ -114,6 +115,8 @@ export default {
       
       let $this = this;
       $this.results = [];
+      $this.temps = []
```

Comment:
Using $this is a code smell. See https://github.com/Shakaii/La-postiterie/pull/6#discussion_r262391344

Reply:
set this instead of $this